GUIDE.md

```diff
@@ -223,29 +240,70 @@ This can be done by adding a `*` to the write access array:
 ```javascript
 const ipfs = new IPFS()
 ipfs.on('ready', async () => {
-  const orbitdb = new OrbitDB(ipfs)
+  const orbitdb = await OrbitDB.createInstance(ipfs)
 
-  const access = {
+  const options = {
     // Give write access to everyone
-    write: ['*'],
+    accessController: {
+      write: ['*']
+    }
   }
 
-  const db = await orbitdb.keyvalue('first-database', access)
+  const db = await orbitdb.keyvalue('first-database', options)
   console.log(db.address.toString())
   // /orbitdb/QmRrauSxaAvNjpZcm2Cq6y9DcrH8wQQWGjtokF4tgCUxGP/first-database
 })
 ```
 
 Note how the access controller hash is different compared to the previous example!
 
+#### Granting access after database creation
+
+To give access to another peer after the database has been created, you must set the access-controller `type` to an `AccessController` which supports dynamically adding keys such as `OrbitDBAccessController`.
+
+```javaScript
+db = await orbitdb1.feed('AABB', {
+  accessController: {
+    type: 'orbitdb', //OrbitDBAccessController
+    write: [id1.publicKey]
+  }
+})
+
+await db.access.grant('write', id2.publicKey) // grant access to id2
+```
+
+#### Custom Access Controller
+
+You can create a custom access controller by implementing the `AccessController` [interface](https://github.com/orbitdb/orbit-db-access-controllers/blob/master/src/access-controller-interface.js) and adding it to the AccessControllers object before passing it to OrbitDB.
+
+```javascript
+class OtherAccessController extends AccessController {
+    static get type () { return 'othertype'} // Return the type for this controller
+    async canAppend(entry, identityProvider) {} // return true if entry can be added, identityProvider allows you to verify entry.identity
```

Comment:
Can we expand on the actual function here? For example:
```javascript
async canAppend(entry, identityProvider) {
  if (entry.payload === "hello world" && entry.identity.id === identity.id)
    return true

  return false
}
```

to give the user a bit more information *how* they could use the canAppend?